views/templates/hook/product-comments-list.tpl

```diff
@@ -27,6 +27,7 @@
   var productCommentAbuseReportErrorMessage = '{l|escape:'javascript' s='Sorry, your abuse report cannot be sent.' d='Modules.Productcomments.Shop'}';
 </script>
 
+<div class="ps-product-comments-container">
 <div class="row">
```

Comment:
Why don't add the `ps-product-comments-container` class on the existing `div` just below?

Reply:
Please, adjust the padding of the child elements

What are you using this class for?